### ISSUE TITLE
[Feature request] Dark theme

### ISSUE THREAD

**Issue by KaKi87** [ziahamza/webui-aria2#604]
Thanks

**Comment by linlinxza** [comment]
I spent some time making a dark style. Not perfect. But here it is. 

Replace the app.css file with with the app.css file in the zip folder under: webui-aria2/docs/
[app.zip](https://github.com/ziahamza/webui-aria2/files/11587646/app.zip)

**Comment by KaKi87** [comment]
Would it be possible to make it user-togglable in settings ?

Thanks

**Comment by linlinxza** [comment]
> Would it be possible to make it user-togglable in settings ?
> 
> Thanks

Nope. I'm not any major software dev. As most, I just know some scripting. I just simply changed color codes in the css file. That is all I can do.
